src/checkpatch_wrapper.sh

```diff
@@ -22,7 +22,7 @@ function execute_checkpatch()
   # output. It could be nice if we can add another option just for this sort
   # of check.
 
-  local options='--no-tree --color=always --strict'
+  local options="${configurations[checkpatch_opts]}"
```

Comment:
What happens if `options` is empty? Do we have a test to check this behavior?

Reply:
I think having no options is a valid usage. If the user clears this option we shouldn't assume any options.